criterion.cabal

```diff
@@ -100,7 +101,7 @@ library
     mwc-random >= 0.8.0.3,
     optparse-applicative >= 0.13,
     parsec >= 3.1.0,
-    statistics >= 0.14 && < 0.16,
+    statistics >= 0.14 && < 0.17,
```

Comment:
Why the major version bump here?

Reply:
Indeed, it isn't necessary as `statistics` worked around GHCJS limitations without major bump.